ts/Series/PolarComposition.ts

```diff
@@ -146,7 +146,7 @@ export declare class PolarSeriesComposition extends Series {
     animate(init?: boolean): void;
     searchPoint: (
         PolarSeriesComposition['kdByAngle'] extends true ?
-            PolarAdditions['searchPointByAngle'] :
+            'searchPointByAngle' :
```

Comment:
`'searchPointByAngle'` is now a string literal type. Try `typeof searchPointByAngle`